### DISCUSSION TITLE
How to edit activity?

### DISCUSSION THREAD

**Discussion by Kathleen1203** [LCA-ActivityBrowser/activity-browser#1326] [Q&A] [answered]
<img width="1347" alt="image" src="https://github.com/user-attachments/assets/6a7acef1-5c54-4aa4-94f0-cf0483e8fe9f">
![Bildschirmfoto 2024-07-13 um 14 18 25](https://github.com/user-attachments/assets/09481660-7a50-4943-a9a8-5199914a01e0)

Hi I want to edit an activity but somehow it is not possible to click on it, or to add a new activity. Could you help me? Thanks in advance!

**Comment by mrvisscher** [comment, marked as answer]
Hi @Kathleen1203 ,

Your database is marked as "read-only" which would explain why you can't edit it. Un-check the read-only checkbox in the database list and you should be good to go!

Kind regards,

Marin
